frontend/src/App.vue

```diff
@@ -1,9 +1,11 @@
 <template>
+
```

Comment:
Would be a good idea to check commits before pushing commits, these changes just add spaces?